core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2794,6 +2794,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_MULTI_RANGE_CHUNK_SIZE_BYTES =
+      new Builder(Name.USER_MULTI_RANGE_CHUNK_SIZE_BYTES)
+          .setDefaultValue(String.format("${%s}", Name.USER_BLOCK_SIZE_BYTES_DEFAULT))
+          .setDescription("Default chunk size for ranged reads from Alluxio files.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
```

Comment:
This is not a client side property, is it?

Reply:
Depends how you look at the term `client`. From the perspective that Alluxio is a client of the underfs then you could consider it `client`. From the "user of Alluxio" perspective I could see it as being a server-sided property.

Reply:
this property is only used in object ufs. ufs is only on the server side.

change alluxio.user => alluxio.underfs.object.store and corresponding property names